modules/ROOT/pages/con_pipeline-image-resources.adoc

```diff
@@ -0,0 +1,91 @@
+[]
+= Image Resource
+
+An Image resource represents an image that lives in a remote repository. It is
+usually used as [a `Task` `output`] for `Tasks` that build
+images. This allows the same `Tasks` to be used to generically push to any
+registry.
+
+Params that can be added are the following:
+
+* `url`: The complete path to the image, including the registry and the image
+    tag
+* `digest`: The
+    [image digest](https://success.docker.com/article/images-tagging-vs-digests)
+    which uniquely identifies a particular build of an image with a particular
+    tag. _While this can be provided as a parameter, there is not yet a way to
+    update this value after an image is built, but this is planned in
+    [#216](https://github.com/tektoncd/pipeline/issues/216)._
+
+For example:
+
+[source]
+----
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: kritis-resources-image
+  namespace: default
+spec:
+  type: image
+  params:
+    - name: url
+      value: gcr.io/staging-images/kritis
+----
+
+
+== Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool
```

Comment:
What exactly do we mean when we say "To surface the image digest"? Since "surface" is an intransitive verb, the builder tool producing some information to "surface" an image digest, just does not sound right. What do you think?
Do we have a better word?